napps/of_cookies.md

```diff
@@ -0,0 +1,18 @@
+---
+title: OpenFlow cookie prefixes
+parent: NApps
+---
+
+OpenFlow cookie prefixes
+========================
+
+Each NApp that sends FlowMod is encouraged to have a configurable and reserved 1-byte cookie prefix to identify which NApp owns a particular flow, to also facilitate queries, debugging and general troubleshooting. This byte is set as the most significant byte.
+
+Currently, the following OpenFlow cookie prefixes have been reserved:
+
+| **NApp**                                           | **OpenFlow cookie prefix / 56** |
+|----------------------------------------------------|---------------------------------|
+| [telemetry](https://github.com/kytos-ng/telemetry) | 0xA8                            |
```

Comment:
FYI @jab1982, here's telemetry's cookie prefix that we've initially discussed. It's also assuming the name of the NApp will be `telemetry` based on blueprint 31.